README.md

```diff
@@ -0,0 +1,60 @@
+# Linux Mint Xed
```

Comment:
While xed is developed primarily by Mint, and was initially forked to meet a need in Mint, it is intended to be cross-de and cross-distro, and is actually available in a number of different distributions and repositories. As such, calling it "Linux Mint Xed" is probably a bit of a misnomer. You might consider calling it "Xed Text Editor" instead.

Reply:
Updated this to now read "Xed Text Editor"